src/statemanager/WfsPermalink.js

```diff
@@ -252,7 +260,7 @@ WfsPermalinkService.prototype.issueRequest_ = function(wfsType, filter, map, sho
     // zoom to features
     const size = map.getSize();
     if (size !== undefined) {
-      const maxZoom = this.pointRecenterZoom_;
+      const maxZoom = zoomLevel || this.pointRecenterZoom_;
```

Comment:
Don't work with zoomLevel == 0...

Reply:
@svaabs Can you change it to this:
`const maxZoom = zoomLevel !== undefined ? zoomLevel : this.pointRecenterZoom_;`